src/Microsoft.Azure.WebJobs.Host.Storage/StorageServiceCollectionExtensions.cs

```diff
@@ -31,17 +31,12 @@ public static void AddAzureStorageCoreServices(this IServiceCollection services)
             // Adds necessary Azure services to create clients
             services.AddAzureClientsCore();
 
-            services.AddAzureStorageScaleServices(null);
+            services.AddAzureStorageScaleServices();
         }
 
-        public static void AddAzureStorageScaleServices(this IServiceCollection services, Action<JobHostInternalStorageOptions> configure)
```

Comment:
Not sure why scaler hosts would need to configure this?

Reply:
This is used in https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host.Storage/AzureStorageProvider.cs#L41. as alternative to `AzureWebJobsStorage`

But I have never seen it was used, is it some legacy thing?